### PR TITLE
Fit 286 implement semantic versioning support for geo services api

### PR DESCRIPTION
**Pull request by mariamcws**
fix for build issue on publishLibraryPublicationToMavenRepository

## Jira Issue link

[Implement Semantic Versioning Support for Geo Services API](https://osi-cwds.atlassian.net/browse/FIT-286)

## Types of changes
- [x] New feature (non-breaking change for continuous integration)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -257,20 +259,19 @@ task pushGitTag(type:Exec, dependsOn: createGitTag) {
     }
 }
 
-processResources {
-    filter(ReplaceTokens, tokens:[
-            'build.version' : projectVersion,
-            'build.number' : (System.getenv("BUILD_NUMBER") ?: "IDE")
-    ])
-}
+//processResources {
```
Comment (LeonidMarushevskyi) [inline build.gradle]: Why this commented. It is needed to populate version so that is available in system-information.

Comment (mariamcws) [reply build.gradle]: still testing it out, not sure why it is breaking the build on jenkins, looks like it works fine locally, I can see that it is populating the version in system-information

File: Jenkinsfile [hunk 2]
```diff
@@ -58,7 +57,10 @@ node ('tpt2-slave'){
    }
    if (env.BUILD_JOB_TYPE=="master" ) {
         stage('Tag Git') {
-           tagGithubRepo(newTag, GITHUB_CREDENTIALS_ID)
+           //tagGithubRepo(newTag, GITHUB_CREDENTIALS_ID)
```
Comment (SergiiRedchuk) [inline Jenkinsfile]: do we need a follow up story to investigate and address differences of tagGithubRepo and this code?

Comment (mariamcws) [reply Jenkinsfile]: this was first cut, next one I will try to use tagGithubRepo(newTag, GITHUB_CREDENTIALS_ID)

**Pull request merged**